lib/super_digit.rb

```diff
@@ -1,15 +1,27 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is number of digits
+# Space Complexity - O(n) because of the call stack
 def super_digit(n)
```

Comment:
This works, but is O(d^2) in time complexity because your summation is O(d) and your recursive method repeats d times.